src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task {
+    public static final char TASK_TYPE = 'D';
+    private final String BY;
```

Comment:
Will need to comply with the Java Coding Standard as variables in class needs to be in camelCase

Consider changing it to "by" instead